homeassistant/components/ecovacs/vacuum.py

```diff
@@ -341,7 +341,7 @@ async def async_send_command(
                 translation_key="vacuum_send_command_params_dict",
             )
 
-        if command in ["spot_area", "custom_area"]:
+        if command in ["spot_area", "custom_area", "update_positions"]:
```

Comment:
We shouldn't add any custom commands to the `send_command` service.
You can use `update_entity` service on the sensor entity instead